chacractl/api/binaries.py

```diff
@@ -136,6 +136,8 @@ def main(self):
                 self.post(url, filepath)
 
         elif self.parser.has('delete'):
+            if self.parser.get('delete') is None:
+                raise SystemExit('Specify a binary URL to delete.')
```

Comment:
can you update this to say: `Specify a URL to delete a binary`  instead?

"binary URL" sounds like something different